linodecli/__init__.py

```diff
@@ -6,21 +6,24 @@
 import argparse
 import os
 import sys
-from importlib import import_module
 from sys import argv
 
 import pkg_resources
-import requests
-import yaml
 from terminaltables import SingleTable
 
 from linodecli import plugins
 
+from .arg_helpers import (
+    action_help,
+    bake_command,
+    help_with_ops,
+    register_args,
+    register_plugin,
+    remove_plugin,
+)
 from .cli import CLI
 from .completion import bake_completions, get_completions
-from .configuration import ENV_TOKEN_NAME
 from .helpers import handle_url_overrides
-from .operation import CLIArg, CLIOperation, URLParam
```

Comment:
Can you undo these deletions? I they may be imported somewhere else in some in-progress work.

Generally, I think we should be careful about the deletions of "unused" imports within the package (deleting unused imports from third party package is less risky, like `requests` and `yaml` in this case) in `__init__.py`.